app/views/pages/help.html.erb

```diff
@@ -118,6 +118,13 @@
 
       <nav role="navigation" class="gem-c-related-navigation__nav-section" aria-labelledby="related-nav-related_items-40a88cdf" data-module="gem-toggle" data-gem-toggle-module-started="true">
         <ul class="gem-c-related-navigation__link-list" data-module="gem-track-click" data-gem-track-click-module-started="true">
+
+          <% unless Rails.env.production? %>
+            <li class="gem-c-related-navigation__link">
+              <%= link_to "Changes 999L", help_changes_999l_path %>
+            </li>
+          <% end %>
```

Comment:
this kind of condition on the FE does not look nice, but I am not sure about an alternative.
Sure this should go only on prod?

Reply:
Its an `unless` so it wont show on prod but actually this doesnt matter. I only had it to show on dev but actually we run rails envs in prod on dev so its pointless. it was literally a very small change that actually needs to be removed by end of day. it didnt work anyways 😂